test/Real.lean

```diff
@@ -0,0 +1,17 @@
+/-
+! This file was ported from Lean 3 source file `test/real.lean`
+! leanprover-community/mathlib commit d7943885503965d07ccaeb390d65fbc3f45962e6
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.Data.Real.Basic
+
+unsafe def testRepr (r : ℝ) (s : String) : Lean.Elab.Command.CommandElabM Unit  :=
+unless toString (repr r) = s do throwError "got {repr r}"
+
+run_cmd testRepr 0 "Real.ofCauchy (sorry /- 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ... -/)"
```

Comment:
This currently fails due to something I don't understand about `unsafe`. I've asked at https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/run_cmd.20and.20unsafe/near/321137546 what the right spelling is. If we finish the port of the rest of the file and this is still pending, we can revert these tests.